other/parse_doc.rb

```diff
@@ -26,7 +26,12 @@ def write_prop(file, node)
   end
   $prop_set[camel_name] += 1
 
-  if name.include? '/'
+  if name.include? 'current-tracks'
+    # According to the mpv documentation this property is not supposed to be
+    # used programmatically. We still output a comment for the property so
+    # people can figure out why the Swift constant is missing.
+    file.write "  /** #{name}  As per mpv doc, scripts etc. should not use this. */\n"
```

Comment:
Nit, I think this reads better:

```suggestion
    file.write "  /** #{name}  As per mpv docs, scripts etc. should not use this. */\n"
```

Reply:
Fixed. I confirmed the issue still exists with parsing the current mpv documentation and a change to the script is still required to avoid compilation errors in the generated code.

I was unsure of the proper workflow for updating a pull request. I'm new to GitHub. I'm used to having to rebase and squash, I read that GitHub has the ability to do that when merging a pull request. Let me know if I need to avoid having a PR with two commits as this one now has.

It is sometimes hard to get a read on people when just communicating through text, so to be real clear I very much appreciate people taking the time to provide a detailed review of anything I'm submitting. Nits and stylistic criticism are very welcome and I am happy to address them. Thank you for all the reviews you are doing.

Reply:
I can't speak to the "right" way to use GitHub, but the way I do it (and we happen to usually work in IINA) is that you keep your branch tidy and we merge that directly, keeping all the commits. What this means in practice is that I will usually make PRs with one or a few commits, each of which that compiles independently; things like "add UI constants" or "implement menu handling". The intent is that each of these can be applied cleanly on top of develop, as if it was incrementally worked on, but it'll come in via a merge.
 
To update my code: if there's a conflict or I've fallen behind develop (but have no changes of my own) I rebase and push. For responses to reviews comments like typos and small suggestions that are obvious I generally just amend and don't bother with a new commit. If the review comments require significant changes and I feel the need to keep around the old implementation for some reason, I'll keep the old commit and push a new one on top.